.github/workflows/cd.yml

```diff
@@ -5,7 +5,64 @@ on:
     branches:
       - main
 
+env:
+  NODE_AUTH_TOKEN: ${{ secrets.PUBLIC_NPM_TOKEN }}
+
```

Comment:
on:
  push:
    branches:
      - main
    paths:
      - 'maasglobal-schema-generator-io-ts'
      - '.github/workflows/deploy-schema-generator-io-ts.yml'

Reply:
Same for other jobs, with the paths changed respectively